snippets/olm-psa-opm-catalog-source-restricted-known-issue.adoc

```diff
@@ -0,0 +1,25 @@
+// Text snippet included in the following assemblies:
```

Comment:
Just a question here on whether this really needs to be added to the actual content?

I thought usually we kept known issues and details to the release notes, and not put throughout applicable docs sections?